openfisca_us/variables/finance/tax/outputs.py

```diff
@@ -142,3 +142,801 @@ class AfterTaxIncome(Variable):
+    entity = TaxUnit
+    definition_period = YEAR
+    documentation = (
+        """search taxcalc/calcfunctions.py for how calculated and used"""
+    )
+
+
+class invinc_agi_ec(Variable):
+    value_type = float
+    entity = TaxUnit
+    definition_period = YEAR
+    documentation = (
+        """search taxcalc/calcfunctions.py for how calculated and used"""
+    )
+
+
+class invinc_ec_base(Variable):
+    value_type = float
+    entity = TaxUnit
+    definition_period = YEAR
+    documentation = (
+        """search taxcalc/calcfunctions.py for how calculated and used"""
+    )
+
+
+class lumpsum_tax(Variable):
+    value_type = float
+    entity = TaxUnit
+    definition_period = YEAR
+    documentation = """Lumpsum (or head) tax; appears as LSTAX variable in tc CLI minimal output"""
+
+
+class pre_c04600(Variable):
+    value_type = float
+    entity = TaxUnit
+    definition_period = YEAR
+    documentation = """Personal exemption before phase-out"""
+
+
+class codtc_limited(Variable):
+    value_type = float
+    entity = TaxUnit
+    definition_period = YEAR
+    documentation = (
+        """search taxcalc/calcfunctions.py for how calculated and used"""
+    )
+
+
+class ptax_amc(Variable):
+    value_type = float
+    entity = TaxUnit
+    definition_period = YEAR
+    documentation = (
+        """Additional Medicare Tax from Form 8959 (included in payrolltax)"""
+    )
+
+
+class ptax_oasdi(Variable):
+    value_type = float
+    entity = TaxUnit
+    definition_period = YEAR
+    documentation = """Employee + employer OASDI FICA tax plus self-employment tax (excludes HI FICA so positive ptax_oasdi is less than ptax_was plus setax)"""
+
+
+class ptax_was(Variable):
+    value_type = float
+    entity = TaxUnit
+    definition_period = YEAR
+    documentation = """Employee + employer OASDI + HI FICA tax"""
+
+
+class setax(Variable):
+    value_type = float
+    entity = TaxUnit
+    definition_period = YEAR
+    documentation = """Self-employment tax"""
+
+
+class ymod(Variable):
+    value_type = float
+    entity = TaxUnit
+    definition_period = YEAR
+    documentation = (
+        """search taxcalc/calcfunctions.py for how calculated and used"""
+    )
+
+
+class ymod1(Variable):
+    value_type = float
+    entity = TaxUnit
+    definition_period = YEAR
+    documentation = (
+        """search taxcalc/calcfunctions.py for how calculated and used"""
+    )
+
+
+class ubi(Variable):
+    value_type = float
+    entity = TaxUnit
+    definition_period = YEAR
+    documentation = """Universal Basic Income benefit for filing unit"""
+
+
+class taxable_ubi(Variable):
+    value_type = float
+    entity = TaxUnit
+    definition_period = YEAR
+    documentation = """Amount of UBI benefit included in AGI"""
+
+
+class nontaxable_ubi(Variable):
+    value_type = float
+    entity = TaxUnit
+    definition_period = YEAR
+    documentation = """Amount of UBI benefit excluded from AGI"""
+
+
+class mtr_paytax(Variable):
```

Comment:
MTRs should probably not be included right?

Reply:
Without having looked in detail at all the variables yet, I assumed it's potentially usable as a variable ([in the UK labour MTRs inform dividend MTRs](https://www.gov.uk/tax-on-dividends))

Reply:
But they're not in `calcfunctions` so removing for now